containers/init/ready/README.md

```diff
@@ -1,25 +1,20 @@
 # Ready
 
-Ready is a container that waits for a list of pods with specific labels to
-become available. It exits successfully when all pods are ready, writing a
+Ready is a container that waits for a list of pods within a load test to
+become available. It exits successfully when all worker pods are ready, writing a
 comma-separated list of their IP addresses to a file. It exits unsuccessfully if
 a timeout was exceeded before all pods were ready.
 
 ## Usage
 
-The container relies on command line arguments to specify the labels pods should
-match. These arguments are in the form of `key=value,key2=value2` where commas
-are treated as `AND`s and spaces delineate separate pods. For example,
+The container relies on command line argument to specify the load test's name.
+For example,
 
 ```shell
-go run ready.go role=server,proxy=envoy role=client role=client
+go run ready.go LOADTEST_NAME
 ```
 
-Will wait for three pods:
-
-1. pod with labels role=server AND proxy=envoy
-2. pod with label role=client
-3. another pod with label role=client
+Will wait for all worker pods which belongs to LOADTEST_NAME
```

Comment:
nit: since there is no longer a list, it may read better if we merge lines 11 and 17:

````suggestion
For example, the following waits for all worker pods which belong to the LoadTest named `<LOADTEST_NAME>`:

```shell
go run ready.go <LOADTEST_NAME>
```
````

Also, this doesn't have to be fixed in this PR (and I apparently broke the convention myself (sorry))... but generally command line arguments adopt the following conventions:

- Angle brackets, such as `<ARG-NAME>`, indicate the argument is required.
- Square brackets, such as `[ARG-NAME]`, indicate the argument is optional.

You will sometimes see no angle brackets, such as the [NetBSD style](https://en.wikipedia.org/wiki/Usage_message#Examples) or the [Google Developer documentation](https://developers.google.com/style/code-syntax). However, the Google developer documentation switches the color and font for required variables. gcloud uses the angle brackets on the command line. If we don't, it can be hard to determine what is a command and what is a required argument. Most of the tools we use rely on angle brackets. For example,

- `git -h`:
  ```
  usage: git [--version] [--help] [-C <path>] [-c <name>=<value>]
           [--exec-path[=<path>]] [--html-path] [--man-path] [--info-path]
           [-p | --paginate | -P | --no-pager] [--no-replace-objects] [--bare]
           [--git-dir=<path>] [--work-tree=<path>] [--namespace=<name>]
           [--super-prefix=<path>] [--config-env=<name>=<envvar>]
           <command> [<args>]
  ```

- `gcloud compute project-info -h`:
  ```
   Usage: gcloud compute project-info [optional flags] <command>
       command may be             add-metadata | describe | remove-metadata |
                                  set-usage-bucket | update
  ```

The Python [docopt](http://docopt.org/) project provides some good conventions. They actually generate a command line argument and flag parser using your usage string documentation.

Reply:
Thanks for providing extra information! Very interesting :)

Reply:
FYI, conventions that are good for usage strings may not be good for markdown documentation. It is not important here because we can always fix it later, so let's merge the change and work on a release.